packages/node_modules/@webex/webex-core/src/webex-core.js

```diff
@@ -544,7 +544,8 @@ const WebexCore = AmpState.extend({
         return p;
       })
       .then((...args) => this._uploadPhaseFinalize(options, ...args))
-      .then((res) => res.body);
+      .then((res) => ({...res.body, ...res.headers}))
+      .catch((err) => err);
```

Comment:
I need to add the trackingId from early on to the error here

Reply:
We are already doing this for the 0 error:

https://github.com/webex/webex-js-sdk/blob/master/packages/node_modules/@webex/http-core/src/request/request.js#L79